### DISCUSSION TITLE
Webview Touch events outside minimised window

### DISCUSSION THREAD

**Discussion by VincentSchuijt-TomTom** [nhn/gpm.unity#401] [Q&A]
Hi, is there any way to get touch events from outside the Webview I just minimsed by me code? Either in unity or in web. Thanks

**Comment by VincentSchuijt-TomTom** [comment]
Or maybe to add, is there some way to make the webview transparent? :D thanks in advance

**Comment by taehyeon-nhn** [comment]
Hello. @VincentSchuijt.

Sorry for the late response.

Currently, WebView cannot obtain external touch events.
We are looking for ways to improve this.
There is no API provided to make the WebView transparent.
We are working on developing the transparency setting for the WebView background color.

Thank you.
